bin/migrate

```diff
@@ -1,2 +1,19 @@
 #!/usr/bin/env ruby
-require_relative("./tasks/migrate.rb")
\ No newline at end of file
+require_relative("./tasks/migrate.rb")
+
+gameobj_xml_path = Pathname.new("./dist/gameobj-data.xml")
+formatted_xml_path = Pathname.new("./dist/gameobj-data.formatted.xml")
+xmllint_cmd = "xmllint --format #{gameobj_xml_path} > #{formatted_xml_path}"
+
+if gameobj_xml_path.exist?
+  print "Formatting XML output with xmllint... "
+  if system(xmllint_cmd) && formatted_xml_path.rename(gameobj_xml_path)
```

Comment:
This will fail if xmllint isn't installed. While this will fail the Travis build, in other scenarios it will mostly just result in an error message.